src/storage/txn/scheduler.rs

```diff
@@ -166,9 +168,15 @@ impl TaskContext {
     }
 
     fn on_schedule(&mut self) {
+        let elapsed = self.latch_timer.saturating_elapsed();
```

Comment:
can we use `TrackGuard` to make code style consistent?

Reply:
My thought is that `TrackGuard` is only used when it's hard to know when we will quit the current function. It may be simpler to get and mutate the tracker directly if it's possible.

Reply:
Got it~ I'm OK with both.